src/LoginScreen/LoginScreen.js

```diff
@@ -66,161 +66,240 @@ class LoginScreen extends Component {
-          className="status-bar">
-          <div className="status-bar-item-1  menu-p" style={{ textAlign: 'left' }}>{window.lightdm.hostname}</div>
+        ref={loginScreen => {
+          this.loginScreen = loginScreen;
+        }}
+      >
+        <div className="status-bar">
+          <div
+            className="status-bar-item-1  menu-p"
+            style={{ textAlign: 'left' }}
+          >
+            {window.lightdm.hostname}
+          </div>
           <div className="status-bar-item-2  menu-p">{this.state.time}</div>
           <div className="status-bar-item-1" style={{ textAlign: 'right' }}>
-          <Dropdown overlay={menu} trigger={['click']}>
-            <span className="power-dropdown menu-p" href="#">
-              <Icon type="poweroff" /> <Icon type="down" />
-            </span>
-          </Dropdown>
+            <Dropdown overlay={menu} trigger={['click']}>
+              <span className="power-dropdown menu-p" href="#">
+                <Icon type="poweroff" /> <Icon type="down" />
+              </span>
+            </Dropdown>
           </div>
         </div>
-        <Form style={{ minWidth: !this.state.isInAuth ? '375px' : '0px', minHeight: !this.state.isInAuth ? '290px' : '0px' }} className="login-form">
+        <Form
+          style={{
+            minWidth: !this.state.isInAuth ? '375px' : '0px',
+            minHeight: !this.state.isInAuth ? '290px' : '0px'
+          }}
+          className="login-form"
+        >
           <Avatar
-            style={{margin: '2rem', fontSize: 48}}
+            style={{ margin: '2rem', fontSize: 48 }}
             size={96}
-            icon={this.state.username ? undefined : "user"}>{this.state.username.toUpperCase()[0]}</Avatar><br/>
-          {!this.state.isInAuth ?
-            ( <Form.Item validateStatus={this.state.validateStatus} help={this.state.warningText}>
+            icon={this.state.username ? undefined : 'user'}
+          >
+            {this.state.username.toUpperCase()[0]}
+          </Avatar>
+          <br />
+          {!this.state.isInAuth ? (
+            <Form.Item
+              validateStatus={this.state.validateStatus}
+              help={this.state.warningText}
+            >
               {this.state.showPassForm ? (
-                  <div>
-                  <span style={{ fontSize: '1.5rem', textAlign: 'left' }} className="font-white">{this.state.username}</span>
+                <div>
+                  <span
+                    style={{ fontSize: '1.5rem', textAlign: 'left' }}
+                    className="font-white"
+                  >
+                    {this.state.username}
+                  </span>
                   <Input.Group compact>
-                    { oPasswordInput }
+                    {oPasswordInput}
                     <Button
                       onClick={() => this.submitPass(this.state.password)}
-                      style={{ width: '15%' }} type="primary"><Icon type="right" /></Button>
```

Comment:
Can you explain why this is being removed?

Reply:
It isn't, it's just being split into several lines for readability